walkthroughs/ecs/regions.md

```diff
@@ -0,0 +1,27 @@
+# AWS App Mesh Region Availability
+
+App Mesh is available in all of the following [AWS regions]:
+
+| AWS Region Name            | Code           |
+|----------------------------|----------------|
+| US West (N. California)    | us-west-1      |
+| US West (Oregon)           | us-west-2      |
+| US East (N. Virginia)      | us-east-1      |
+| US East (Ohio)             | us-east-2      |
+| Canada (Central)           | ca-central-1   |
+| Europe (Ireland)           | eu-west-1      |
+| Europe (London)            | eu-west-2      |
+| Europe (Stockholm)         | eu-north-1     |
+| Europe (Frankfurt)         | eu-central-1   |
+| Asia Pacific (Mumbai)      | ap-south-1     |
+| Asia Pacific (Singapore)   | ap-southeast-1 |
+| Asia Pacific (Sydney)      | ap-southeast-2 |
+| Asia Pacific (Tokyo)       | ap-northeast-1 |
+| Asia Pacific (Seoul)       | ap-northeast-2 |
+| Asia Pacific (Osaka-Local) | ap-northeast-3 |
+| China (Beijing)            | cn-north-1     |
+| China (Ningxia)            | cn-northwest-1 |
+| AWS GovCloud (US-East)     | us-gov-east-1  |
+| AWS GovCloud (US-West)     | us-gov-west-1  |
+
+[AWS regions]: https://github.com/aws/aws-app-mesh-roadmap/issues/1
```

Comment:
A few things here:

1. App Mesh is not available in all of these regions. See the issue you linked at the bottom for the accurate list (**checked** boxes only).
2. Does it make sense to move this to the main repository readme?

Reply:
It's missing a 3rd column that had Xs for the actual supported regions, but I'm going to remove this altogether and just point to the official link to check.

Reply:
Regarding making it the main README, @kiranmeduri also suggested that and I'm fine with that. I'll make the change and push an update shortly.